__tests__/__datasets__/path-matching-quirks.json

```diff
@@ -1,5 +1,5 @@
 {
-  "openapi": "3.0.0",
+  "openapi": "3.1.0",
```

Comment:
Changed this to 3.1 because I didn't feel like adding responses to these two new paths.